bootstrap.sh

```diff
@@ -24,13 +24,15 @@ usage()
     echo "Usage: $0 [-h] [+docs]"
     echo "Bootstrap ReFrame by pulling all its dependencies"
     echo "  -P EXEC  Use EXEC as Python interpreter"
+    echo "  -O       Install optional packages using pip"
```

Comment:
I would rather prefer `+pygelf` option.

Reply:
I did like that in case we need to add more packages in the feature.

Reply:
I know, but I prefer to restrict it and be more explicit. I don't think we will be adding random packages very often.